doc/api/v0.yaml

```diff
@@ -56,6 +68,35 @@ paths:
               schema:
                 $ref: "#/components/schemas/Proposal"
 
+  /api/v0/challenges:
+    get:
+      summary: Get all available challenges
+      responses:
+        200:
+          description: Valid response
+          content:
+            application/json:
+              schema:
+                items:
+                  $ref: "#/components/schemas/Challenge"
+
+  /api/v0/challenges/{id}:
+    get:
+      summary: Get challenges by id
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: integer
+          required: true
+      responses:
```

Comment:
Could also document a 404 status if the challenge is not found. We do it in at least four places in this spec, yay diligence.